src/modules/rating/services/rating-scheduler.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable, Logger } from '@nestjs/common'
+import { Cron } from '@nestjs/schedule'
+import * as dotenv from 'dotenv'
+import { RatingRepository } from '../database/rating.repository'
+
+@Injectable()
+export class RatingScheduler {
+	private readonly logger = new Logger(RatingScheduler.name)
+
+	constructor(private readonly ratingRepository: RatingRepository) {
+		dotenv.config()
+	}
+
+	@Cron(process.env.CRON_SCHEDULE)
+	async deleteExpiredRejectedRatings() {
+		try {
+			const expiredTime = new Date(
+				Date.now() - parseInt(process.env.EXPIRED_TIME) * 1000,
```

Comment:
Don't use the environment variable directly from `process.env`. We should inject it via a service called **ConfigService**. Checkout this module to know how to implement this:
- https://github.com/patecan-io-software/trn-ecom-phucuong-inventory/blob/509d674eefbde6808815bd16e54177aeb873c46e/src/modules/admin/auth/auth.controller.ts#L24
- NestJS config service document: https://docs.nestjs.com/techniques/configuration